src/renderer/cursor_renderer/mod.rs

```diff
@@ -270,6 +271,7 @@ impl CursorRenderer {
             )
                 .into();
         }
+        if self.destination != prev_destination {}
```

Comment:
It looks like some code is missing here.

Reply:
Not, missing, but an extra branch that I forgot to remove, when I implemented the IME cursor tracking differently.

I will delete it.

Reply:
It's deleted now.